app/src/main/java/org/mifos/mobile/ui/fragments/GuarantorDetailFragment.kt

```diff
@@ -113,19 +93,19 @@ import javax.inject.Inject
     override fun onOptionsItemSelected(item: MenuItem): Boolean {
         when (item.itemId) {
             R.id.menu_delete_guarantor -> MaterialDialog.Builder()
-                    .init(context)
-                    .setTitle(R.string.delete_guarantor)
-                    .setMessage(getString(R.string.dialog_are_you_sure_that_you_want_to_string,
-                            getString(R.string.delete_guarantor)))
-                    .setPositiveButton(getString(R.string.yes),
-                            DialogInterface.OnClickListener { _, _ ->
-                                presenter?.deleteGuarantor(loanId, guarantorId)
-                            })
-                    .setNegativeButton(R.string.cancel)
-                    .createMaterialDialog()
-                    .show()
+                .init(context)
+                .setTitle(R.string.delete_guarantor)
+                .setMessage(getString(R.string.dialog_are_you_sure_that_you_want_to_string,
+                    getString(R.string.delete_guarantor)))
+                .setPositiveButton(getString(R.string.yes),
+                    DialogInterface.OnClickListener { _, _ ->
```

Comment:
This redundant SAM constructor can be reformat. Please have a look.

Reply:
done